data/variabledefs-rescue.csv

```diff
@@ -39,11 +39,12 @@ CO2,Solvents Production and Application,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2
 CO2,Transportation Sector,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,TRA
 CO2,Waste,Mt CO2/yr,TRUE,country,anthro_CO2.nc,CO2_em_anthro,WST
 CO2,CDR Afforestation,Mt CO2/yr,TRUE,global,CDR_CO2.nc,CO2_em_anthro,A/R
-CO2,CDR BECCS,Mt CO2/yr,TRUE,global,CDR_CO2.nc,CO2_em_anthro,BECCS
+CO2,CDR BECCS,Mt CO2/yr,TRUE,region,CDR_CO2.nc,CO2_em_anthro,BECCS
 CO2,CDR DACCS,Mt CO2/yr,TRUE,country,CDR_CO2.nc,CO2_em_anthro,DAC_CDR
 CO2,CDR EW,Mt CO2/yr,TRUE,country,,,
 CO2,CDR Industry,Mt CO2/yr,TRUE,country,CDR_CO2.nc,CO2_em_anthro,IND_CDR
 CO2,CDR OAE Uptake Ocean,Mt CO2/yr,TRUE,region,CDR_CO2.nc,CO2_em_anthro,OAE_CDR
+CO2,Ocean Alkalinity,,FALSE,region,CDR_CO2.nc,CO2_em_anthro,OAE_CDR

```

Comment:
Should this gridded ocean alkalinity be part of the CO2_em_anthro file? What is the unit?

Reply:
per our discussion:

- variable name -> 'Alkalinity Addition'
- yes to the co2 anthro file (no better place atm)
- unit should come from @merfort and be provided as a flux